getting_started/setup_linux/prepare_build_environment.rst

```diff
@@ -0,0 +1,104 @@
+.. _prepare-build-environment:
+
+Set up a development environment to build the kernel
+####################################################
+
+.. contents::
+   :local:
+   :depth: 3
+
+Introduction
+************
+
+
+These instructions will help you set up a development environment for the SOF branch of the Linux kernel. If you have dedicated test hardware you can use ktest to install it over ssh, otherwise you can install it locally on your device in addition to your default kernel.
+
+Prerequisites
+*************
+
+1. Device requirements
+-----------------------
+
+**Development device:** PC running Fedora 35+ or Ubuntu 20.04+.
+
+**Target device:** PC running Fedora 35+ or Ubuntu 20.04+, with secure boot disabled. If the target device is different than the development device you must be able to ssh into the target, which is typically on the same local network/VPN.
+
+2. Create working directory
+---------------------------
+
+This can be located anywhere, but later commands assume its path is ~/sof
+It will host all the sources you need to get
+
+.. code-block:: bash
+   
+   mkdir ~/sof
+   cd ~/sof
```

Comment:
The SOF section defines a `SOF_WORKSPACE=~/work/sof` indirection so users can always copy/paste the commands. Would be good to do this here too for consistency and to have everything in the same top-level directory.


PS: I think `~/sof/` sucks because it leads to a double `~/sof/sof/`

Reply:
In this case it doesn't lead to `~/sof/sof`, as the repository is cloned to `linux`. I'll still change it to use the `SOF_WORKSPACE` anyways, that's a good idea.

Reply:
> In this case it doesn't lead to ~/sof/sof, as the repository is cloned to linux. 

The idea is to have the same SOF_WORKSPACE for both kernel and firmware.

> I'll still change it to use the SOF_WORKSPACE anyways, that's a good idea.

Thanks!